apps/prairielearn/python/prairielearn.py

```diff
@@ -41,7 +41,7 @@ class PartialScore(TypedDict):
 
     score: float | None
     weight: NotRequired[int]
-    feedback: NotRequired[str]
+    feedback: NotRequired[str | dict[str, str]]
```

Comment:
I'm inclined to say that this should just be `Any`; we don't place any restrictions on what this can be.

Reply:
Changed in 0f3dbb5ef2cac4153f51a8d3e8d536b56d0e0dfb. I kept the type hints for well-known feedback values.

Reply:
What's the rationale for a type like `str | dict[str, str] | Any`? Just as a form of documentation? I'd be surprised if a type-checker could do anything useful with that.

Reply:
Yeah, a form of documentation.

Reply:
I'm not at all sold on the value, as feedback is truly element-dependent. I won't block on this.

Reply:
I personally find it helpful when my editor shows a typehint when I am coding, even if it isn't picked up by the typechecker. If it isn't too much trouble I would like to keep it like this.